### ISSUE TITLE
Feature request: FPS counter?

### ISSUE THREAD

**Issue by chenglou** [markusmoenig/ShaderMania#6]
Thanks for this project! =)

Would it be possible to display the FPS of the running shader?

**Comment by markusmoenig** [comment]
Yes sure, actually I had one already implemented but displaying the FPS counter in the Swift UI is very very slow. I need to make the extra effort to render the text inside a shader and display it in the preview which is a bit of a hassle.
